app/src/assets/localization/en/shared.json

```
@@ -6,6 +6,7 @@
   "browse": "browse",
   "cancel": "cancel",
   "clear_data": "clear data",
+  "close_robot_door": "Close the robot door before starting the run",
```

```suggestion
  "close_robot_door": "Close the robot door before starting the run.",
```
Nit, seems to match the format of other full sentences in the file.